test/test_easy_optimisation.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+#
+# Tests the easy optimisation methods fmin and curve_fit.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2018, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import pints
+import unittest
+#import numpy as np
+
+
+class TestEasyOptimisation(unittest.TestCase):
+    """
+    Tests the easy optimisation methods fmin and curve_fit.
+    """
+    def test_fmin(self):
+        """
+        Tests :meth:`pints.fmin()`.
+        """
+        # Note: This just wraps around `Optimisation`, so testing done here is
+        # for wrapper code, not main functionality!
+
+        # Basic test
+        def f(x):
+            return (x[0] - 3) ** 2 + (x[1] + 5) ** 2
+        xopt, fopt = pints.fmin(f, [1, 1], method=pints.XNES)
+        self.assertAlmostEqual(xopt[0], 3)
+        self.assertAlmostEqual(xopt[1], -5)
+
+        # Function must be callable
+        self.assertRaisesRegexp(ValueError, 'callable', pints.fmin, 3, [1])
```

Comment:
curious as to why this is here. Does it really just test if the function is a function (i.e. can be called)?

Reply:
It's a test to see if the correct error is raised if `f` isn't callable! (It should raise a ValueError with some error message containing the word "callable").